auth/checkToken.js

```diff
@@ -0,0 +1,19 @@
+const token = require( 'jsonwebtoken' )
+require( 'dotenv' ).config()
```

Comment:
We should have access to process.env here already though.  This would only be needed if this code were executing outside of the server environment, I think?